modules/tinymce/src/plugins/image/main/ts/ui/MainTab.ts

```diff
@@ -28,7 +28,7 @@ const makeItems = (info: ImageDialogInfo): Dialog.BodyComponentSpec[] => {
     name: 'alt',
     type: 'input',
     label: 'Alternative description',
-    disabled: info.hasAccessibilityOptions && info.image.isDecorative
+    enabled: !info.hasAccessibilityOptions || !info.image.isDecorative
```

Comment:
```suggestion
    enabled: !(info.hasAccessibilityOptions && info.image.isDecorative)
```
This is closer to what we're trying to say here, so it reads a little better I think 🤷

Reply:
🤷